src/datumaro/plugins/data_formats/datumaro/base.py

```diff
@@ -23,90 +26,42 @@
 from datumaro.components.dataset_base import DatasetItem, SubsetBase
 from datumaro.components.errors import DatasetImportError, MediaTypeError
 from datumaro.components.importer import ImportContext
-from datumaro.components.media import Image, MediaElement, PointCloud
+from datumaro.components.media import Image, MediaElement, MediaType, PointCloud
 from datumaro.util import parse_json_file
 from datumaro.version import __version__
 
 from .format import DATUMARO_FORMAT_VERSION, DatumaroPath
 
 
-class DatumaroBase(SubsetBase):
-    LEGACY_VERSION = "legacy"
-    CURRENT_DATUMARO_FORMAT_VERSION = DATUMARO_FORMAT_VERSION
-
-    # If Datumaro format version goes up, it will be
-    # ALLOWED_VERSIONS = {LEGACY_VERSION, 1.0, ..., CURRENT_DATUMARO_FORMAT_VERSION}
-    ALLOWED_VERSIONS = {LEGACY_VERSION, CURRENT_DATUMARO_FORMAT_VERSION}
-
+class DefaultReader:
```

Comment:
How about renaming this by `DatumaroJsonReader` if you have any plan to introduce similar JsonReader for other formats like COCO. If not, just `JsonReader` and `StreamJsonReader` sound good to me.

Reply:
Rename them in https://github.com/openvinotoolkit/datumaro/pull/1082/commits/8d507974ab8f08d3f51d55d566bada4fe9268b90